core/server/middleware/theme-handler.js

```diff
@@ -78,9 +79,6 @@ themeHandler = {
 
         blogApp.engine('hbs', hbs.express3(hbsOptions));
 
-        // Update user error template
-        errors.updateActiveTheme(activeTheme);
```

Comment:
i hope that helps
https://github.com/TryGhost/Ghost/pull/7431/commits/114cc12feaf32b9214f515747137a7dbcdde62ea